Sources/TSCBasic/Process.swift

```diff
@@ -471,10 +472,26 @@ public final class Process {
                 pathString: ProcessEnv.path,
                 currentWorkingDirectory: cwdOpt
             )
+            var searchPaths: [AbsolutePath] = []
+#if os(Windows)
+            var buffer = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
+            // The 32-bit Windows system directory
+            if GetSystemDirectoryW(&buffer, .init(buffer.count)) > 0 {
+                searchPaths.append(AbsolutePath(String(decodingCString: buffer, as: UTF16.self)))
+            }
+            if GetWindowsDirectoryW(&buffer, .init(buffer.count)) > 0 {
+                let windowsDirectory = String(decodingCString: buffer, as: UTF16.self)
+                // The 16-bit Windows system directory
+                searchPaths.append(AbsolutePath("\(windowsDirectory)\\System"))
+                // The Windows directory
+                searchPaths.append(AbsolutePath(windowsDirectory))
+            }
+#endif
+            searchPaths.append(contentsOf: envSearchPaths)
```

Comment:
Hmm, I think that this should be inverted no?  We want the environment to take precedent don't we?

Reply:
I followed the behavior of `CreateProcess`, where these three directories are always preferred.
But it’s reasonable for TSC to not respect that, though this shouldn’t have any real impact.

Reply:
Hmm, I think that folowing `CreateProcess` is reasonable.